Yoast/wordpress-seo
# Move redirect modal to premium

## Summary

This PR can be summarized in the following changelog entry:

## Relevant technical choices:

* Remove AJAX handling of saving a redirect.
* Remove the creation/existing redirect modal.
* Add a modal object to generate the modal contents.
* Remove the JS for sending the redirect.

## Test instructions

This PR can be tested by following these steps:

* Do a `grunt build:js`. 
* Check if the modal opens on the GSC page -> when it shows a no premium modal it still works.
* Check if no JS errors occur.

This pull is needed for: https://github.com/Yoast/wordpress-seo-premium/issues/579
